src/CoreBundle/Resources/public/javascript/table.js

```diff
@@ -117,4 +141,8 @@ $(document).ready(function () {
     });
 
     CHAMELEON.CORE.initializeEntryAutocomplete($('input#searchLookup'));
+
+    var $searchFields = $("input[name='_search_word']");
+    $searchFields.on("keypress", resetPagingAndSearch); // NOTE "input" is not called for a RETURN (which immediately sends the form)
+    $searchFields.closest("form").find("input[type='button']").on("click", resetPagingAndSearch);
```

Comment:
Same point as above. we do not want to add the event to just any form on the page - but only to a full group table form. 
I would suggest adding a class to clearly mark it and then selecting based on that